README.md

```diff
@@ -8,60 +8,122 @@
 
-The library does not include any Provider/inject utilities as they can be fully replaced with [React Context](https://mobx-react.js.org/recipes-context). Check out [the migration guide](https://mobx-react.js.org/recipes-migration).
+| mobx | mobx-react-lite | mobx-react<br/>(for class components) | Browser                                        |
+| ---- | --------------- | ------------------------------------- | ---------------------------------------------- |
+| 6    | 3               | 7                                     | Modern browsers (IE 11+ in compatibility mode) |
+| 5    | 2               | 6                                     | Modern browsers                                |
+| 4    | 2               | 6                                     | IE 11+, RN w/o Proxy support                   |
 
-Class based components **are not supported** except using `<Observer>` directly in class `render` method. If you want to transition existing projects from classes to hooks, use [mobx-react 6+](https://github.com/mobxjs/mobx-react).
-
-See more at [the libraries overview](https://mobx-react.js.org/libraries).
+`mobx-react-lite` requires React 16.8 or higher.
 
 ## User Guide 👉 https://mobx.js.org/react/react-integration.html
 
-The site contains various examples and recipes for using MobX in React world. Feel free to contribute. The API reference of this package follows 👇.
+---
 
 ## API reference ⚒
 
-### **`observer<P>(baseComponent: FunctionComponent<P>, options?: IObserverOptions): FunctionComponent<P>`**
-
-```ts
-interface IObserverOptions {
-    // Pass true to wrap the inner component with React.forwardRef.
-    // It's false by the default.
-    forwardRef?: boolean
-}
-```
+### **`observer<P>(baseComponent: FunctionComponent<P>): FunctionComponent<P>`**
 
 The observer converts a component into a reactive component, which tracks which observables are used automatically and re-renders the component when one of these values changes.
+Can only be used for function components. For class component support see the `mobx-react` package.
 
 ### **`<Observer>{renderFn}</Observer>`**
 
-Is a React component, which applies observer to an anonymous region in your component.
+Is a React component, which applies observer to an anonymous region in your component. `<Observer>` can be used both inside class and function components.
+
+### **`useLocalObservable<T>(initializer: () => T, annotations?: AnnotationsMap<T>): T`**
+
+Creates an observable object with the given properties, methods and computed values.
+
+Note that computed values cannot directly depend on non-observable values, but only on observable values, so it might be needed to sync properties into the observable using `useEffect` (see the example below at `useAsObservableSource`).
+
+`useLocalObservable` is a short-hand for:
+
+`const state = useState(() => observable(initializer(), annotations, { autoBind: true }))[0]`
+
+### **`useStaticRendering(enable: true)`**
```

Comment:
I think we should deprecate this name and find something that doesn't look like a hook :) Something like `enableStaticRendering` or even `enableSSR`.